default.nix

```diff
@@ -128,6 +129,33 @@ rec {
     import ./templates/project { inherit lib pkgs sources; }
   );
 
+  pre-commit-hook = pre-commit-hooks.run {
+    src = ./.;
+    hooks = {
+      actionlint.enable = true;
+      nixfmt-rfc-style.enable = true;
+      actionlint.package = pkgs.actionlint;
+      nixfmt-rfc-style.package = pkgs.nixfmt-rfc-style;
+    };
+  };
+
+  formatter =
+    {
+      pre-commit ? pre-commit-hook,
+    }:
+    pkgs.writeShellApplication {
+      name = "formatter";
```

Comment:
Would be good to add this to the devshell, so one can just run something like `format` be done with it. It would be a way better UX than with `nix fmt` because that always takes forever to evaluate the underlying expression.